app/services/ValidateMetadataService.scala

```diff
@@ -0,0 +1,29 @@
+package services
+
+import configuration.GraphQLConfiguration
+import play.api.libs.ws.WSClient
+import play.api.{Configuration, Logging}
+
+import java.util.UUID
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ValidateMetadataService @Inject() (val ws: WSClient, val configuration: Configuration, graphQLConfiguration: GraphQLConfiguration)(implicit
+    val executionContext: ExecutionContext
+) extends Logging {
+
+  def triggerMetadataValidation(consignmentId: UUID, token: String): Future[Boolean] = {
+    val url = s"${configuration.get[String]("metadatavalidation.baseUrl")}/draft-metadata/validate/$consignmentId"
```

Comment:
Can you use the `ApplicationConfig` to get the `metadatavalidation.baseUrl`? You need to add a variable for that:

  `val metadataValidationBaseUrl: String = configuration.get[String]("metadatavalidation.baseUrl")
`